### PR TITLE
fix docs api

### PR DESCRIPTION
**Pull request by rnyak**

### CODE REVIEW

File: docs/source/examples/outbrains.ipynb [hunk 1]
```diff
@@ -1 +0,0 @@
-../../../examples/wnd_outbrain/E2E-NVT.ipynb
```
Comment (benfred) [inline docs/source/examples/outbrains.ipynb:1]: I don't think we can delete this - did you mean to rename from outbrains.ipynb -> outbrain.ipynb?

File: nvtabular/workflow.py [hunk 2]
```diff
@@ -41,11 +41,11 @@ class BaseWorkflow:
 
     Parameters
     -----------
-    cat_names : list of str
```
Comment (benfred) [inline nvtabular/workflow.py:44]: We shouldn't make this change - 'list of str' is correct here. See the numpy docstring guide
https://numpydoc.readthedocs.io/en/latest/format.html#sections (search for 'list of str').

**Review by benfred — CHANGES_REQUESTED** [review]

File: docs/source/examples/outbrains.ipynb (hunk 1 above)
Comment (rnyak) [reply docs/source/examples/outbrains.ipynb:1]: Yes.

Comment (rnyak) [reply docs/source/examples/outbrains.ipynb:1]: @benfred cant we remove E2E-NVT.ipynb and add a new notebook named `Outbrain.ipynb`? I will create a PR for outbrain notebook, and I want to name the notebook as `Outbrain.ipynb`.

File: README.md [hunk 3]
```diff
@@ -28,10 +28,10 @@ Our goal is faster iteration on massive tabular datasets, both for experimentati
 ### Getting Started
 NVTabular is available in the NVIDIA container repository at the following location: http://ngc.nvidia.com/catalog/containers/nvidia:nvtabular.
 
-Currently we have the beta release (0.2) container, you can pull the container using the following command:
+Currently we have the beta release (0.3) container, you can pull the container using the following command:
```
Comment (benfred) [inline README.md:31]: We don't have a container up at this link yet - lets remove this change for now, and make this change when we have published the new NGC container

File: docs/source/examples/outbrains.ipynb (hunk 1 above)
Comment (benfred) [reply docs/source/examples/outbrains.ipynb:1]: yes - we can change the filename of the notebook

We can't delete this symlink entirely though, it's needed to render the outbrain notebook in the docs. Once you've changed the filename can you add a new symlink to the file (also changing the symlinks name from outbrains -> outbrain?)

**Review by benfred — APPROVED** [review]

**Comment by benfred** [comment]
rerun tests

**Comment by benfred** [comment]
rerun tests

**Comment by benfred** [comment]
rerun tests

**Comment by benfred** [comment]
rerun tests

**Comment by benfred** [comment]
rerun tests

**Comment by benfred** [comment]
rerun tests

**Pull request merged**